recipes/mdsrv/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "MDsrv" %}
+{% set version = "0.3.5" %}
+{% set sha256 = "3dd218ec0d62cf47061699b8ab33715d6875fad824091f2ab37e546467fef0c3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install 
+
+requirements:
+  build:
+    - python
+    - cython
+    - numpy >=1.11
+    - msinttypes  # [win and py2k]
+    - setuptools
+    - flask
+    - scipy
+    - zlib
+    - mdtraj
+    - mdanalysis  # [not win and py2k]
+
+  run:
+    - python
+    - setuptools
+    - numpy >=1.11
+    - flask
+    - mdtraj
+    - mdanalysis  # [not win and py2k]
+
+test:
+  imports:
```

Comment:
Please test the command line interface:

https://conda.io/docs/building/meta-yaml.html#test-commands